ansible/aws/gather_facts.yaml

```diff
@@ -0,0 +1,11 @@
+- name: Gather system facts
+  hosts: all
+  gather_facts: yes
+  tasks:
+    - name: Get disk details
+      shell: lsblk -o FSTYPE,SIZE,UUID,PATH,MOUNTPOINT -r | grep 'ext4\|xfs'
```

Comment:
can we be sure that the filesystem will be ext4 or xfs always?

Reply:
I modelled it after the feature for windows.

Reply:
I modelled it on what the current footprint does to filter disks.